kokoro/scripts/utils/louhi.sh

```diff
@@ -28,14 +28,18 @@ function populate_env_vars_from_louhi_tag_if_present() {
   # Example value: louhi/2.46.0/shortref/windows/x86_64/start
   if [[ -n "${_LOUHI_TAG_NAME:-}" ]]; then
     local -a _LOUHI_TAG_COMPONENTS=(${_LOUHI_TAG_NAME//\// })  
-    export REPO_SUFFIX="${_LOUHI_TAG_COMPONENTS[2]}"  # the shortref is the repo suffix
+    RELEASE_ID="${_LOUHI_TAG_COMPONENTS[2]}"
     TARGET="${_LOUHI_TAG_COMPONENTS[3]}"
     ARCH="${_LOUHI_TAG_COMPONENTS[4]}"
-    export ARTIFACT_REGISTRY_PROJECT="${_STAGING_ARTIFACTS_PROJECT_ID}"  # Louhi is responsible for passing this.
 
-    EXT=$(yaml project.yaml "['targets']['${TARGET}']['package_extension']")
-    if [[ "${EXT}" == "deb" ]]; then
-      export REPO_CODENAME="${TARGET}-${ARCH}"
+    if [[ "${INSTALL_AGENT_UNDER_TEST:-1}" == "1" ]]; then
```

Comment:
[Optional] I wonder if inverting this flag to, e.g., `INSTALL_RELEASED_AGENT_VERSION` would be more readable? But I won't let it block the PR if you choose to not do that.
```suggestion
    if [[ "${INSTALL_RELEASED_AGENT_VERSION:-}" == "1" ]]; then
```

Reply:
I also thought on this for a while. I leaned the way I did because the flag matches the behavior it gates, rather than the implicit behavior of the test (install the released version.)